### DISCUSSION TITLE
Multiple BME280

### DISCUSSION THREAD

**Discussion by Bannhammer** [theyosh/TerrariumPI#960] [Q&A] [answered]
Hello Folks!
I've been building a terrarium Pi for a few days now.
I use BME280 sensors as temperature sensors. 
Now I would like to connect 2 of these sensors via the I2C bus.
For this I got a multiplexer (PCA9548A).
When I connect the multiplexer (VIN,GRND,SDA,SCA) the Pi also shows me the address (70) of the multiplexer when I enter i2cdetect -y 1. But now when I connect a sensor the Pi can no longer find the address of the sensor.
Does anyone here happen to have a solution to the problem?

**Comment by theyosh** [comment, marked as answer]
Does this helps? https://theyosh.github.io/TerrariumPI/hardware/i2c-multiplexer/

In short: Make the OS control the multiplexer. And you should have than multiple I2C busses. So use `i2cdetect -y 3` etc

**Comment by Bannhammer** [comment]
Thanks alot Yosh! It worked :)

**Reply by theyosh** [reply]
Nice!

**Comment by diablomaster9045** [comment]
Another option for this is ground out the SDO pin and it will drop the address to 0x76.
